cdap-standalone/src/main/java/co/cask/cdap/StandaloneMain.java

```diff
@@ -69,6 +69,8 @@
 import co.cask.cdap.metrics.query.MetricsQueryService;
 import co.cask.cdap.notifications.feeds.guice.NotificationFeedServiceRuntimeModule;
 import co.cask.cdap.notifications.guice.NotificationServiceRuntimeModule;
+import co.cask.cdap.operations.OperationalStatsLoader;
```

Comment:
These two classes should move out of the cdap-operational-states module. Are they needed by cdap-master as well? If that's the case, either put them in common or need a new module for that. The master/standalone modules shouldn't depend on the `cdap-operational-stats` directly.